Formula/c/codesmooth.rb

```diff
@@ -0,0 +1,15 @@
+class Codesmooth < Formula
+  desc "Command-line interface for CodeSmooth tasks"
+  homepage "https://codesmooth.dev"
+  url "https://storage.googleapis.com/codesmooth-resources/public/cli/codesmooth-1.0.0.tar.gz"
```

Comment:
if you just distribute the binary, it would be a good fit for cask.

Reply:
ok, moving it to cask repo